### PR TITLE
Updated agent to bot and changed code to match requirements

### PR DESCRIPTION
**Pull request by Tdrachev**
Updated code to match hasseb's suggestions

### CODE REVIEW

**Review by dikel — APPROVED** [review]
LGTM

File: forta-core-monitoring/src/agent-scanner-minted.js [hunk 1]
```diff
@@ -19,21 +19,24 @@ function handleScannerMintingTransfer(
...
-          severity: FindingSeverity.Low,
-          type: FindingType.Info,
-          metadata: {
-            to,
-            tokenId: tokenIdNormalized,
-          },
-        })
-      );
+    if (tokenId && from && to) {
+      const tokenIdNormalized = ethers.BigNumber.from(tokenId).toString();
```
Comment (haseebrabbani) [inline forta-core-monitoring/src/agent-scanner-minted.js:23]: will this be decimal value or hex value?

File: forta-core-monitoring/src/agent.config.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "abi": [
     "event Transfer(address indexed from, address indexed to, uint256 indexed tokenId)",
-    "event StakeThresholdChanged(uint256 min, uint256 max, bool activated)",
+    "event StakeThresholdChanged(uint256 indexed chainId, uint256 min, uint256 max, bool activated)",
```
Comment (haseebrabbani) [inline forta-core-monitoring/src/agent.config.json:4]: on second look, there is a different StakeThresholdChanged event defined for agent registry contract vs scanner registry contract (only the scanner event has the chainId parameter). we probably want to monitor events for both agents and scanners

**Review by stoqnkpL — APPROVED** [review]
LGTM

**Pull request merged**